test/lib/ansible_test/_util/controller/sanity/code-smell/shebang.py

```diff
@@ -83,6 +84,10 @@ def main():
                     print('%s:%d:%d: should not have a shebang' % (path, 0, 0))
 
                 continue
+            elif re.search('^roles/[^/]+/(?:files|templates)/', path):
+                is_role_file = True
+            elif re.search('^tests?/integration/targets/[^/]+/(?:files|templates)/', path):
```

Comment:
This change shouldn't affect core, only collections:

```suggestion
            elif re.search('^tests/integration/targets/[^/]+/(?:files|templates)/', path):
```